db/migrate/20230725135855_add_gillick_competence_details_to_consent_response.rb

```diff
@@ -1,4 +1,6 @@
-class AddGillickCompetenceDetailsToConsentResponse < ActiveRecord::Migration[7.0]
+class AddGillickCompetenceDetailsToConsentResponse < ActiveRecord::Migration[
```

Comment:
🤔 this feels like a weird change...

Reply:
It's trying to wrap the lines to 80 characters where it can; that's 78 long

Reply:
aha, gotcha